go.sum

```diff
@@ -337,10 +337,14 @@ github.com/keep-network/keep-common v1.2.1-0.20201020114759-19c123cbd4f4 h1:Civu
 github.com/keep-network/keep-common v1.2.1-0.20201020114759-19c123cbd4f4/go.mod h1:emxogTbBdey7M3jOzfxZOdfn139kN2mI2b2wA6AHKKo=
 github.com/keep-network/keep-common v1.2.1-0.20201112133003-054c43201b4b h1:ErsUI/oSwXPUHxNq7j/ttZXn3WOn4bOCf2QJoo2gvZE=
 github.com/keep-network/keep-common v1.2.1-0.20201112133003-054c43201b4b/go.mod h1:emxogTbBdey7M3jOzfxZOdfn139kN2mI2b2wA6AHKKo=
+github.com/keep-network/keep-common v1.2.1-0.20201116121154-016d77aacdce h1:B+kei2KjTdLS7JeK6YeDFU6suzPbkCn8WiAX1VDqtA0=
+github.com/keep-network/keep-common v1.2.1-0.20201116121154-016d77aacdce/go.mod h1:emxogTbBdey7M3jOzfxZOdfn139kN2mI2b2wA6AHKKo=
 github.com/keep-network/keep-core v1.3.0 h1:7Tb33EmO/ntHOEbOiYciRlBhqu5Ln6KemWCaYK0Z6LA=
 github.com/keep-network/keep-core v1.3.0/go.mod h1:1KsSSTQoN754TrFLW7kLy50pOG2CQ4BOfnJqdvEG7FA=
 github.com/keep-network/tbtc v1.1.1-0.20201026093513-cb9246987718 h1:/ZNMBY7y6hfzCYA8mgtHnspGO26OmWV3sDehyGnqRyY=
 github.com/keep-network/tbtc v1.1.1-0.20201026093513-cb9246987718/go.mod h1:igBF2MPTFkzOdZ3gcwt8h0Zb5pZaHnij/iPZoMB9IKM=
+github.com/keep-network/tbtc v1.1.1-0.20201116122054-001d3ad0cfc6 h1:p3t1G0aXXl2dZ+hVRFwSlPqXdFr3TW/dcYtKN9u79ds=
+github.com/keep-network/tbtc v1.1.1-0.20201116122054-001d3ad0cfc6/go.mod h1:OTDuVugn5XnKPpKgWNDgD8Ziv4PTUZ9mKM8k40JSl1w=
```

Comment:
Let's remember to `go mod tidy` at the end.

Reply:
Done